jedi_language_server/jedi_utils.py

```diff
@@ -72,6 +72,7 @@ def lsp_range(name: Name) -> Range:
 
 def lsp_location(name: Name) -> Location:
     """Get LSP location from Jedi definition."""
+    assert name.module_path is not None
     return Location(uri=name.module_path.as_uri(), range=lsp_range(name))
```

Comment:
```suggestion
def lsp_location(name: Name) -> Optional[Location]:
    """Get LSP location from Jedi definition."""
    return (
        Location(uri=name.module_path.as_uri(), range=lsp_range(name))
        if name.module_path is not None
        else None
    )
```

I'd prefer that we return an `Optional[Location]` here so that Jedi's `module_path` quirks are properly reflected in our types instead of our potential exceptions.